kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -331,31 +332,31 @@ private void incrementPaymentCount() {
 	}
 
 	private void startPaymentListener() {
-		paymentRegistration = account.blockchainEvents()
-			.addPaymentListener(new EventListener<PaymentInfo>() {
-				@Override
-				public void onEvent(PaymentInfo data) {
-					final String orderID = extractOrderId(data.memo());
-					Logger.log(new Log().withTag(TAG).put("startPaymentListener onEvent: the orderId", orderID)
-						.put("with memo", data.memo()));
-					final String accountPublicAddress = getPublicAddress();
-					if (orderID != null && accountPublicAddress != null) {
-						completedPayment.postValue(PaymentConverter.toPayment(data, orderID, accountPublicAddress));
-						Logger.log(new Log().withTag(TAG).put("completedPayment order id", orderID));
-					}
-					// UpdateBalance if there is no balance sse open connection.
-					if (balanceObserversCount == 0) {
-						final double prevBalance = balance.getValue().getAmount().doubleValue();
-						getBalance(new KinCallbackAdapter<Balance>() {
-							@Override
-							public void onResponse(Balance response) {
-								eventLogger.send(KinBalanceUpdated.create(prevBalance));
-							}
-						});
+			paymentRegistration = account.blockchainEvents()
+				.addPaymentListener(new EventListener<PaymentInfo>() {
+					@Override
+					public void onEvent(PaymentInfo data) {
+						final String orderID = extractOrderId(data.memo());
+						Logger.log(new Log().withTag(TAG).put("startPaymentListener onEvent: the orderId", orderID)
+							.put("with memo", data.memo()));
+						final String accountPublicAddress = getPublicAddress();
+						if (orderID != null && accountPublicAddress != null) {
+							completedPayment.postValue(PaymentConverter.toPayment(data, orderID, accountPublicAddress));
+							Logger.log(new Log().withTag(TAG).put("completedPayment order id", orderID));
+						}
+						// UpdateBalance if there is no balance sse open connection.
+						if (balanceObserversCount == 0) {
+							final double prevBalance = balance.getValue().getAmount().doubleValue();
+							getBalance(new KinCallbackAdapter<Balance>() {
+								@Override
+								public void onResponse(Balance response) {
+									eventLogger.send(KinBalanceUpdated.create(prevBalance));
+								}
+							});
+						}
 					}
-				}
-			});
-	}
+				});
+		}
```

Comment:
Same comment as above - format the code because i think you have an extra tab

Reply:
:+1: